pkg/spireentry/reconciler.go

```diff
@@ -110,11 +110,16 @@ func (r *entryReconciler) reconcile(ctx context.Context) {
 		}
 
 		clusterSPIFFEID.NextStatus.Stats.NamespacesSelected += len(namespaces)
+		var ignored = false
+
 		for i := range namespaces {
-			if r.config.IgnoreNamespaces.In(namespaces[i].Name) {
+			ignored = checkIgnoredNamespace(r.config.IgnoreNamespaces, namespaces[i].Name)
+
+			if ignored {
```

Comment:
The ignored variable is used in a single spot. I'm curious if the code isn't more clear if we just use the function call directly in the if statement. It might help readability if we rename the function slightly:
```suggestion
			if isNamespaceIgnored(r.config.IgnoreNamespaces, namespaces[i].Name) {
```